src/cryptography/x509.py

```diff
@@ -237,6 +237,57 @@ def __repr__(self):
                 "path_length={0.path_length})>").format(self)
 
 
+class KeyUsage(object):
+    def __init__(self, usages):
```

Comment:
(Of note, the indices of the list match the bitmask index values in RFC 5280, although that's certainly just an implementation detail)